Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DccStateChecker.kt

```diff
@@ -45,7 +45,7 @@ internal fun DccData<*>.getExpirationState(
     val daysUntilExpiration = now.daysUntil(expiresAt, timeZone)
     val diff = daysUntilExpiration - expirationThresholdInDays
     return when {
-        daysUntilExpiration <= 0 -> CwaCovidCertificate.State.Expired(expiresAt)
+        daysUntilExpiration < 0 -> CwaCovidCertificate.State.Expired(expiresAt)
```

Comment:
So you want to add the case == 0 and check the timestamp to decide whether it is already expired